From 9af889d07e0f26489055631b378834246b477388 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Santiago=20Ruano=20Rinc=C3=B3n?= <santiagorr@riseup.net>
Date: Thu, 23 Jul 2026 12:35:21 -0300
Subject: [PATCH 07599/15333] staging: rtl8723bs: fix several line spaces in
 wifi.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Address checkpatch.pl warnings fixing reports of type LINE_SPACING:

WARNING: Missing a blank line after declarations
CHECK: Please don't use multiple blank lines
CHECK: Please use a blank line after function/struct/union/enum
declarations

Signed-off-by: Santiago Ruano Rincón <santiagorr@riseup.net>
Reviewed-by: Nikolay Kulikov <nikolayof23@gmail.com>
Link: https://patch.msgid.link/20260723153526.255965-1-santiagorr@riseup.net
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/rtl8723bs/include/wifi.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/staging/rtl8723bs/include/wifi.h b/drivers/staging/rtl8723bs/include/wifi.h
index d5132bc4280a..4bf158d429c6 100644
--- a/drivers/staging/rtl8723bs/include/wifi.h
+++ b/drivers/staging/rtl8723bs/include/wifi.h
@@ -163,7 +163,6 @@ enum {
 #define SetDuration(pbuf, dur) \
 	(*(__le16 *)((size_t)(pbuf) + 2) = cpu_to_le16(0xffff & (dur)))
 
-
 #define SetPriority(pbuf, tid)	\
 	(*(__le16 *)(pbuf) |= cpu_to_le16(tid & 0xf))
 
@@ -192,12 +191,15 @@ enum {
 static inline unsigned char *rtl8723bs_get_ra(unsigned char *pframe)
 {
 	unsigned char *ra;
+
 	ra = GetAddr1Ptr(pframe);
 	return ra;
 }
+
 static inline unsigned char *get_ta(unsigned char *pframe)
 {
 	unsigned char *ta;
+
 	ta = GetAddr2Ptr(pframe);
 	return ta;
 }
@@ -225,7 +227,6 @@ static inline unsigned char *get_da(unsigned char *pframe)
 	return da;
 }
 
-
 static inline unsigned char *get_sa(unsigned char *pframe)
 {
 	unsigned char *sa;
@@ -272,11 +273,11 @@ static inline unsigned char *get_hdr_bssid(unsigned char *pframe)
 	return sa;
 }
 
-
 static inline int IsFrameTypeCtrl(unsigned char *pframe)
 {
 	return GetFrameType(pframe) == WIFI_CTRL_TYPE;
 }
+
 /*-----------------------------------------------------------------------------
 			Below is for the security related definition
 ------------------------------------------------------------------------------*/
@@ -335,7 +336,6 @@ struct ieee80211_ht_addt_info {
 	unsigned char basic_set[16];
 } __packed;
 
-
 struct HT_caps_element {
 	union {
 		struct {
-- 
2.55.0

